app/models/comment.rb

```diff
@@ -0,0 +1,9 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: :author_id
+  belongs_to :post
```

Comment:
The foreign key is improperly implemented. As per the given ERD, the Comment association should have Comment belong to User through Author, an alias of User, like below:
`belongs_to :author, class_name: 'User'`, the `foreign_key: :author_id` is not necessary.

Kindly ensure to improve upon this aspect.